inst/@sym/solve.m

```diff
@@ -31,9 +31,9 @@
 %% @example
 %% @group
 %% syms x
-%% solve(x == 2*x + 6, x)
+%% solve (x == 2 * x + 6, x)
```

Comment:
okis, to know, what operators need spaces and what not?
https://www.gnu.org/software/octave/doc/v4.0.3/Operator-Index.html